packages/system-tests/package.json

```diff
@@ -15,7 +15,7 @@
     "@types/node": "^6.0.40",
     "@types/rimraf": "0.0.28",
     "@types/shelljs": "^0.7.4",
-    "@types/webdriverio": "^4.6.1",
+    "@types/webdriverio": "4.6.1",
```

Comment:
@chowwinston, @Sweetman, @GaryFNC, @DatGuyJonathan - Just FYI. When I compile this locally, I saw some type errors with TypeScript 2.4.0 if I let it use the next (that is what the ^ signifies) version of types. So, I'm locking it down to 4.6.1 for now.